tests/pdk/finfet/test_circuits.py

```diff
@@ -85,6 +85,8 @@ def test_cmp_2():
         {"constraint": "GroupBlocks", "instances": ["mp5", "mp6"], "name": "ccp"},
         {"constraint": "GroupBlocks", "instances": ["mn11", "mp13"], "name": "invp"},
         {"constraint": "GroupBlocks", "instances": ["mn12", "mp14"], "name": "invn"},
+        {"constraint": "SameTemplate", "instances": ["mp9","mp10"]},
```

Comment:
@soneryaldiz Added these constraints to make sure the same concrete instance name is used/

Reply:
Thanks Steve. It would be nice to make this the default behavior (i.e., if abstract names are same for a symmetry pair, add same template constraint). Else, we would need this for mp7 & mp8, invn & invp as well.